lib/vector-core/src/transform/mod.rs

```diff
@@ -239,7 +239,7 @@ impl TransformOutputs {
 
 async fn send_inner(buf: &mut Vec<Event>, output: &mut Fanout) {
     for event in buf.drain(..) {
-        output.feed(event).await.expect("unit error");
+        output.send(event).await.expect("unit error");
     }
```

Comment:
Since this one is bounded, I think we'd be better off doing `feed` in the loop and then one `flush` at the end.

Reply:
That's a good point. 👍🏻

Reply:
Updated, better?